src/imdb_detector.py

```diff
@@ -127,6 +138,15 @@ def is_not_a_production_year(self, value: str) -> bool:
         if not tokens[1].isdigit():
             return tokens[1]
         return 0
+    
+    def is_not_a_cast_person_role_id(self, value):
+        try:
+            value = float(value)
+            if value != 999999.0:
+                return 1
+            return 0
+        except ValueError:
+            return 1
```

Comment:
Please always return the value itself, never `1` or `True`. This can lead to errors in the specific labeling code, which are very hard to detect, because we are using 1 then instead of the wrong value for detection. The case for 0 can stay like this.